features/FEATURE_BLE/targets/TARGET_STM/TARGET_NUCLEO_WB55RG/stm32wb_HCIDriver.cpp

```diff
@@ -0,0 +1,902 @@
+/*
```

Comment:
General comment because I'm not going to point out each line but code style is inconsistent, I assume copy&paste from existing code with a different style. The braces and indent change and the sections are mixed - this could really do with splitting up into a header and cpp file (separate for driver and transport driver) - make this more maintainable in the long run.

Reply:
Ok sure. I will try to improve code style consistency. 
Do you have a reference implementation of separate driver and transport driver that can be used as a good example?

Reply:
https://github.com/ARMmbed/mbed-os/tree/master/features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NORDIC_CORDIO/TARGET_NRF5x

Reply:
Thanks.
About code style, note that the Porting guide template is maybe not fully consistent with mbed coding style ...  https://github.com/armmbed/mbed-os/blob/master/features/FEATURE_BLE/targets/TARGET_CORDIO/doc/PortingGuide.md

Reply:
thanks, that's embarrassing, I'll make a note to fix that

Reply:
that's indeed the code I started from ... ;-)